spec/query.spec.js

```diff
@@ -655,16 +655,26 @@ describe('Query on indexed BigInts #141', () => {
                 votes: BigInt(movie.votes),
             };
         });
-        await moviesRef.set(ObjectCollection.from(movies));
+        // await moviesRef.set(ObjectCollection.from(movies));
+        await db.indexes.create('movies', 'title');
+        await db.indexes.create('movies', 'rating');
         await db.indexes.create('movies', 'votes');
+
+        movies.forEach( async (movie) => {
```

Comment:
`forEach` is not async. Each movie is added in your async callback, but `forEach` does not await your callbacks.

Reply:
If you want to be sure all adds are awaited before unit tests run, change the code to
```js
        for (const movie of movies) {
            await moviesRef.push(movie);
        }
```